scripts/cypress-ci.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable no-console */
+
+const BoxSDK = require('box-node-sdk');
+const childProcess = require('child_process');
+
+const { env } = process;
+const { E2E_ACCESS_TOKEN, E2E_CLIENT_ID, E2E_PARENT_FOLDER_ID } = env;
+
+const sdk = new BoxSDK({
+    clientID: E2E_CLIENT_ID,
+    clientSecret: 'none',
+});
+
+const client = sdk.getBasicClient(E2E_ACCESS_TOKEN);
+
+async function main() {
+    const testFolderName = `Test ${new Date().toISOString()}`;
+
+    console.log(`Setup test folder: ${testFolderName}...`);
+    const { id: folderId } = await client.folders.create(E2E_PARENT_FOLDER_ID, testFolderName);
+    const { id: documentId } = await client.files.copy('694470903390', folderId); // Document template
+    const { id: imageId } = await client.files.copy('694468799644', folderId); // Image template
```

Comment:
Should these be env variables too?

Reply:
Each will require its own line item in the script, so that didn't seem necessary.